docker/envfiles/local.env

```diff
@@ -0,0 +1,6 @@
+DEBUG=False
+DEV=False
+ALLOWED_HOSTS=*
+ADMINS=["thedude@example.com"]
+REDIS_URL=redis://redis:6379
+WAGTAIL_ENABLE_ADMIN: True
```

Comment:
This will be redefined by the root `.env` that should have the values one would like to use where needed, so just a naïve Q: do the `DEV`+`DEBUG` have a reason to be `False` instead of `True` here? Is it so that the worker job itself shouldn't need to print out any debug info per se?

Reply:
Yeah, I guess it could work in Debug mode locally. My default is to keep things strict in order to avoid surprises, unless there's a need to relax it, but happy to tweak.

Reply:
Actually, thinking about this a bit more, keeping it stricter by default will mean running `release-local` and the local queue behave as in production. AFAICT `DEBUG` doesn't directly influence our logging levels, but does make Django's internals behave differently upon error, and I'd rather - by default - see prod-like problems than debug-mode versions of them. As you say, this will be overridden by anything in an on-filesystem `.env` file